src/amp-story-player/amp-story-player-impl.js

```diff
@@ -985,28 +985,34 @@ export class AmpStoryPlayer {
   /**
    * Navigates stories given a number.
    * @param {number} storyDelta
+   * @param {number=} pageDelta
    */
-  go(storyDelta) {
-    if (storyDelta === 0) {
+  go(storyDelta, pageDelta = 0) {
+    if (storyDelta === 0 && pageDelta === 0) {
       return;
     }
```

Comment:
Nit: this is not necessary because it will short-circuit in other places when both are 0, but it's also fine to leave as is